src/main/java/mate/academy/dao/impl/CinemaHallDaoImpl.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.dao.impl;
+
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.CinemaHall;
+import mate.academy.model.Movie;
```

Comment:
The import statement for `mate.academy.model.Movie` is unnecessary as the `Movie` class is not used in this file. Consider removing this import to clean up the code.